libminifi/test/unit/C2MetricsTests.cpp

```diff
@@ -32,22 +32,25 @@ TEST_CASE("TestProcessMetrics", "[c2m1]") {
   minifi::state::response::ProcessMetrics metrics;
 
   REQUIRE("ProcessMetrics" == metrics.getName());
-
+#ifndef WIN32
   REQUIRE(2 == metrics.serialize().size());
```

Comment:
metrics don't exist on windows. should create a ticket to eventually resolve this.